contract_deployer/Cargo.toml

```diff
@@ -16,7 +16,11 @@ ethers = { tag = "v2.0.15-fix-reconnections", features = [
 mina-curves = { git = "https://github.com/openmina/proof-systems", branch = "ledger-newtypes-rampup4-vrf" }
 o1-utils = { git = "https://github.com/lambdaclass/proof-systems", branch = "add-verifier-serializations" }
 kimchi = { git = "https://github.com/openmina/proof-systems", branch = "ledger-newtypes-rampup4-vrf" }
-aligned-sdk = { git = "https://github.com/lambdaclass/aligned_layer.git", branch = "mina" }
+aligned-sdk = { git = "https://github.com/lambdaclass/aligned_layer.git", branch = "account_inclusion_verifier" }
```

Comment:
This dependency should point to `mina` branch (we can change it once we merge [the PR #22 of Aligned fork](https://github.com/lambdaclass/aligned_layer/pull/22))

Reply:
Merged it! will change the dependency now